### PR TITLE
chore: add variable descriptions

### PR DESCRIPTION
**Pull request by zhpenkov**
Adds variable descriptions to the variables of the following components:

- Splitter
- Wizard
- Stepper
- OrgChart
- Chat
- TaskBoard

### CODE REVIEW

File: packages/bootstrap/scss/stepper/_variables.scss [hunk 1]
```diff
@@ -1,87 +1,224 @@
...
+/// @group stepper
 $kendo-stepper-indicator-disabled-bg: null !default;
+/// The text color of the disabled Stepper indicator.
+/// @group stepper
 $kendo-stepper-indicator-disabled-text: if($kendo-enable-color-system, rgba( k-color( on-app-surface, true ), .65 ), $kendo-disabled-text) !default;
+/// The border color of the disabled Stepper indicator.
+/// @group stepper
 $kendo-stepper-indicator-disabled-border: null !default;
 
+/// The background color of the Stepper done indicator.
```
Comment (yordan-mitev) [inline packages/bootstrap/scss/stepper/_variables.scss:111]: Adding the `'s` for clarity here.
```suggestion
/// The background color of the Stepper's done indicator.
```

File: packages/bootstrap/scss/stepper/_variables.scss [hunk 2]
```diff
@@ -1,87 +1,224 @@
...
+/// @group stepper
 $kendo-stepper-indicator-done-bg: $kendo-color-primary !default;
+/// The text color of the Stepper done indicator.
```
Comment (yordan-mitev) [inline packages/bootstrap/scss/stepper/_variables.scss:114]:
```suggestion
/// The text color of the Stepper's done indicator.
```

File: packages/bootstrap/scss/stepper/_variables.scss [hunk 3]
```diff
@@ -1,87 +1,224 @@
...
+/// @group stepper
 $kendo-stepper-indicator-done-text: if($kendo-enable-color-system, k-color( on-primary ), k-contrast-color( $kendo-stepper-indicator-done-bg )) !default;
+/// The border color of the Stepper done indicator.
```
Comment (yordan-mitev) [inline packages/bootstrap/scss/stepper/_variables.scss:117]:
```suggestion
/// The border color of the Stepper's done indicator.
```

File: packages/bootstrap/scss/stepper/_variables.scss [hunk 4]
```diff
@@ -1,87 +1,224 @@
...
+/// @group stepper
 $kendo-stepper-indicator-done-border: $kendo-stepper-indicator-done-bg !default;
 
+/// The background color of the hovered Stepper done indicator.
```
Comment (yordan-mitev) [inline packages/bootstrap/scss/stepper/_variables.scss:121]:
```suggestion
/// The background color of the Stepper's hovered done indicator.
```

File: packages/bootstrap/scss/stepper/_variables.scss [hunk 5]
```diff
@@ -1,87 +1,224 @@
...
+/// @group stepper
 $kendo-stepper-indicator-done-hover-bg: if($kendo-enable-color-system, k-color( primary-hover ), k-try-shade( $kendo-stepper-indicator-done-bg )) !default;
+/// The text color of the hovered Stepper done indicator.
```
Comment (yordan-mitev) [inline packages/bootstrap/scss/stepper/_variables.scss:124]:
```suggestion
/// The text color of the Stepper's hovered done indicator.
```

File: packages/bootstrap/scss/stepper/_variables.scss [hunk 6]
```diff
@@ -1,87 +1,224 @@
...
+/// @group stepper
 $kendo-stepper-indicator-done-hover-text: null !default;
+/// The border color of the hovered Stepper done indicator.
```
Comment (yordan-mitev) [inline packages/bootstrap/scss/stepper/_variables.scss:127]:
```suggestion
/// The border color of the Stepper's hovered done indicator.
```

File: packages/bootstrap/scss/stepper/_variables.scss [hunk 7]
```diff
@@ -1,87 +1,224 @@
...
+/// @group stepper
 $kendo-stepper-indicator-done-hover-border: null !default;
 
+/// The background color of the disabled Stepper done indicator.
```
Comment (yordan-mitev) [inline packages/bootstrap/scss/stepper/_variables.scss:131]:
```suggestion
/// The background color of the Stepper's disabled done indicator.
```

File: packages/bootstrap/scss/stepper/_variables.scss [hunk 8]
```diff
@@ -1,87 +1,224 @@
...
+/// @group stepper
 $kendo-stepper-indicator-done-disabled-bg: if($kendo-enable-color-system, rgba( k-color( primary, true ), .6 ), k-color-mix( $kendo-stepper-indicator-done-bg, $kendo-component-bg, 60%)) !default;
+/// The text color of the disabled Stepper done indicator.
```
Comment (yordan-mitev) [inline packages/bootstrap/scss/stepper/_variables.scss:134]:
```suggestion
/// The text color of the Stepper's disabled done indicator.
```

File: packages/bootstrap/scss/stepper/_variables.scss [hunk 9]
```diff
@@ -1,87 +1,224 @@
...
+/// @group stepper
 $kendo-stepper-indicator-done-disabled-text: if($kendo-enable-color-system, k-color( on-primary ), k-contrast-color( $kendo-stepper-indicator-done-bg )) !default;
+/// The border color of the disabled Stepper done indicator.
```
Comment (yordan-mitev) [inline packages/bootstrap/scss/stepper/_variables.scss:137]:
```suggestion
/// The border color of the Stepper's disabled done indicator.
```

File: packages/bootstrap/scss/stepper/_variables.scss [hunk 10]
```diff
@@ -1,87 +1,224 @@
...
+/// @group stepper
 $kendo-stepper-indicator-current-bg: $kendo-stepper-indicator-done-bg !default;
+/// The text color of the Stepper current indicator.
+/// @group stepper
 $kendo-stepper-indicator-current-text: $kendo-stepper-indicator-done-text !default;
+/// The border color of the Stepper current indicator.
+/// @group stepper
 $kendo-stepper-indicator-current-border: $kendo-stepper-indicator-done-border !default;
 
+/// The background color of the hovered Stepper current indicator.
```
Comment (yordan-mitev) [inline packages/bootstrap/scss/stepper/_variables.scss:151]:
```suggestion
/// The background color of the Stepper's hovered current indicator.
```

File: packages/bootstrap/scss/stepper/_variables.scss [hunk 11]
```diff
@@ -1,87 +1,224 @@
...
+/// @group stepper
 $kendo-stepper-indicator-current-hover-bg: $kendo-stepper-indicator-done-hover-bg !default;
+/// The text color of the hovered Stepper current indicator.
```
Comment (yordan-mitev) [inline packages/bootstrap/scss/stepper/_variables.scss:154]:
```suggestion
/// The text color of the Stepper's hovered current indicator.
```

File: packages/bootstrap/scss/stepper/_variables.scss [hunk 12]
```diff
@@ -1,87 +1,224 @@
...
+/// @group stepper
 $kendo-stepper-indicator-current-hover-text: $kendo-stepper-indicator-done-hover-text !default;
+/// The border color of the hovered Stepper current indicator.
```
Comment (yordan-mitev) [inline packages/bootstrap/scss/stepper/_variables.scss:157]:
```suggestion
/// The border color of the Stepper's hovered current indicator.
```

File: packages/bootstrap/scss/stepper/_variables.scss [hunk 13]
```diff
@@ -1,87 +1,224 @@
...
+/// The border width of the focused Stepper indicator.
+/// @group stepper
 $kendo-stepper-indicator-focus-border-width: 1px !default;
+/// The size of the focused Stepper indicator.
+/// @group stepper
 $kendo-stepper-indicator-focus-size: 2px !default;
+/// The offset of the Stepper focused indicator.
+/// @group stepper
 $kendo-stepper-indicator-focus-offset: 3px !default;
+/// The calculated offset of the focused Stepper indicator.
```
Comment (yordan-mitev) [inline packages/bootstrap/scss/stepper/_variables.scss:77]:
```suggestion
/// The calculated offset of the Stepper's focused indicator.
```

File: packages/bootstrap/scss/stepper/_variables.scss [hunk 14]
```diff
@@ -1,87 +1,224 @@
...
+/// @group stepper
 $kendo-stepper-indicator-current-hover-bg: $kendo-stepper-indicator-done-hover-bg !default;
+/// The text color of the hovered Stepper current indicator.
+/// @group stepper
 $kendo-stepper-indicator-current-hover-text: $kendo-stepper-indicator-done-hover-text !default;
+/// The border color of the hovered Stepper current indicator.
+/// @group stepper
 $kendo-stepper-indicator-current-hover-border: $kendo-stepper-indicator-done-hover-border !default;
 
+/// The background color of the disabled Stepper current indicator.
```
Comment (yordan-mitev) [inline packages/bootstrap/scss/stepper/_variables.scss:161]:
```suggestion
/// The background color of the Stepper's disabled current indicator.
```

File: packages/bootstrap/scss/stepper/_variables.scss [hunk 15]
```diff
@@ -1,87 +1,224 @@
...
+/// @group stepper
 $kendo-stepper-indicator-current-disabled-bg: $kendo-stepper-indicator-done-disabled-bg !default;
+/// The text color of the disabled Stepper current indicator.
```
Comment (yordan-mitev) [inline packages/bootstrap/scss/stepper/_variables.scss:164]:
```suggestion
/// The text color of the Stepper's disabled current indicator.
```

File: packages/bootstrap/scss/stepper/_variables.scss [hunk 16]
```diff
@@ -1,87 +1,224 @@
...
+/// @group stepper
 $kendo-stepper-indicator-current-disabled-text: $kendo-stepper-indicator-done-disabled-text !default;
+/// The border color of the disabled Stepper current indicator.
```
Comment (yordan-mitev) [inline packages/bootstrap/scss/stepper/_variables.scss:167]:
```suggestion
/// The border color of the Stepper's disabled current indicator.
```

File: packages/bootstrap/scss/stepper/_variables.scss [hunk 17]
```diff
@@ -1,87 +1,224 @@
...
+/// @group stepper
 $kendo-stepper-label-error-text: $kendo-color-error !default;
+/// The text color of the hovered Stepper label.
+/// @group stepper
 $kendo-stepper-label-hover-text: null !default;
+/// The text color of the disabled Stepper label.
+/// @group stepper
 $kendo-stepper-label-disabled-text: if($kendo-enable-color-system, rgba( k-color( on-app-surface, true ), .65 ), $kendo-disabled-text) !default;
 
+/// The text color of the Stepper optional label.
```
Comment (yordan-mitev) [inline packages/bootstrap/scss/stepper/_variables.scss:187]:
```suggestion
/// The text color of the optional Stepper label.
```

File: packages/bootstrap/scss/stepper/_variables.scss [hunk 18]
```diff
@@ -1,87 +1,224 @@
...
+/// @group stepper
 $kendo-stepper-optional-label-text: $kendo-subtle-text !default;
+/// The opacity of the Stepper optional label.
```
Comment (yordan-mitev) [inline packages/bootstrap/scss/stepper/_variables.scss:190]:
```suggestion
/// The opacity of the optional Stepper label.
```

File: packages/bootstrap/scss/stepper/_variables.scss [hunk 19]
```diff
@@ -1,87 +1,224 @@
...
+/// @group stepper
 $kendo-stepper-optional-label-opacity: null !default;
+/// The font size of the Stepper optional label.
```
Comment (yordan-mitev) [inline packages/bootstrap/scss/stepper/_variables.scss:193]:
```suggestion
/// The font size of the optional Stepper label.
```

File: packages/bootstrap/scss/stepper/_variables.scss [hunk 20]
```diff
@@ -1,87 +1,224 @@
...
+/// @group stepper
 $kendo-stepper-optional-label-font-size: $kendo-font-size-sm !default;
+/// The font style of the Stepper optional label.
```
Comment (yordan-mitev) [inline packages/bootstrap/scss/stepper/_variables.scss:196]:
```suggestion
/// The font style of the optional Stepper label.
```

File: packages/bootstrap/scss/taskboard/_variables.scss [hunk 21]
```diff
@@ -1,94 +1,245 @@
...
+/// The vertical padding of the TaskBoard column header.
+/// @group taskboard
 $kendo-taskboard-column-header-padding-y: $kendo-taskboard-spacer * .75 !default;
+/// The horizontal padding of the TaskBoard column header.
+/// @group taskboard
 $kendo-taskboard-column-header-padding-x: $kendo-taskboard-column-header-padding-y !default;
+/// The spacing of the TaskBoard column header.
+/// @group taskboard
 $kendo-taskboard-column-header-gap: k-math-div( $kendo-taskboard-spacer, 4 ) !default;
+/// The spacing  of the TaskBoard column header actions.
```
Comment (yordan-mitev) [inline packages/bootstrap/scss/taskboard/_variables.scss:108]:
```suggestion
/// The spacing of the TaskBoard column header actions.
```

File: packages/bootstrap/scss/wizard/_variables.scss [hunk 22]
```diff
@@ -1,13 +1,34 @@
...
+/// The vertical padding of the Wizard.
+/// @group wizard
 $kendo-wizard-padding-y: $kendo-wizard-padding-x !default;
+/// The horizontal padding of the Wizard content.
+/// @group wizard
 $kendo-wizard-content-padding-x: $kendo-padding-md-x * 2 !default;
+/// The vertical padding of the Wizard content.
+/// @group wizard
 $kendo-wizard-content-padding-y: $kendo-padding-md-y * 4 !default;
+/// The horizontal margin of the Wizard Button container.
```
Comment (yordan-mitev) [inline packages/bootstrap/scss/wizard/_variables.scss:15]:
```suggestion
/// The horizontal margin of the Wizard's Button container.
```

**Review by Juveniel — APPROVED** [review]

**Pull request merged**